.github/actionlint-matcher.json

```diff
@@ -0,0 +1,17 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "actionlint",
+      "pattern": [
+        {
+          "regexp": "^(?:\\x1b\\[\\d+m)?(.+?)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*: (?:\\x1b\\[\\d+m)*(.+?)(?:\\x1b\\[\\d+m)* \\[(.+?)\\]$",
```

Comment:
what is this voodoo?

Reply:
A [GitHub Problem Matcher](https://github.com/actions/toolkit/blob/main/docs/problem-matchers.md) file for [ActionLint](https://github.com/rhysd/actionlint)

ActionLint lints our GitHub actions workflows, and the problem matcher highlights problems that were found in the "Files Changed" view of the PR, just like how the presubmits in Critique show where in the code the problem was.